test/BreadcrumbItemSpec.js

```diff
@@ -137,10 +137,30 @@ describe('<Breadcrumb.Item>', () => {
   });
 
   it('Should spread property on customized inner link element', () => {
-    const instance = mount(<Breadcrumb.Item linkAs={Button} type="submit" />);
+    const instance = mount(
+      <Breadcrumb.Item linkAs={Button} linkProps={{ type: 'submit' }} />,
+    );
     instance
       .find('button')
       .prop('type')
       .should.eq('submit');
   });
+
+  it('Should be able to pass attributes to the link element', () => {
+    const instance = mount(
+      <Breadcrumb.Item linkProps={{ foo: 'bar' }}>Crumb</Breadcrumb.Item>,
+    );
+    instance
+      .find('a')
+      .prop('foo')
+      .should.eq('bar');
+  });
+
+  it('Should be able to pass attributes to the li element', () => {
+    const instance = mount(<Breadcrumb.Item foo="bar">Crumb</Breadcrumb.Item>);
+    instance
+      .find('li')
+      .prop('foo')
+      .should.eq('bar');
+  });
```

Comment:
Could we re-write these tests to make assertions on actual use cases?

Reply:
you mean dont use `should.eq`?
I adjusted the test names